app/js/reducers/addDrugOrderReducer.js

```diff
@@ -0,0 +1,27 @@
+import {
+  POST_DRUG_ORDER_SUCCESS,
+  POST_DRUG_ORDER_FAILURE,
+} from '../../../app/js/actions/actionTypes';
+
+const initialState = {
+  addedOrder: {},
+  error: null,
+};
+
```

Comment:
Shouldn't the initial state read from the mockdata, so that new orders can be added to ```activeOrders```, in order for the component to update the active orders table?

Reply:
Okay